### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by tmk1176**
changed

### CODE REVIEW

**Comment by tmk1176** [comment]
pls review the changes and let me to know about it.

**Comment by prawinpatil** [comment]
yes ..i will check and let u know about it

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # hello-world
 demo
+hi....this is change made by me: tnk1176.
```
Comment (prawinpatil) [inline README.md]: pls check the spelling it must be tmk not tnk.

Comment (tmk1176) [reply README.md]: ok

**Review by prawinpatil — COMMENTED** [review]
ok...let me check

**Pull request merged**